tests/ut/Readme.md

```diff
@@ -0,0 +1,29 @@
+# Azure IoT Middleware Unit Test Framework
+
+## Overview
+
+The files in this directory implement the device SDK portion of unit testing of the Embedded SDK for Azure IoT Convenience layer.

```

Comment:
```suggestion
The files in this directory implement the device SDK portion of unit testing of the Azure IoT middleware for FreeRTOS.

```

Reply:
Good catch! Copy/paste. Fixing in both places.